chibuezeayogu/React-Redux
# #161155295 Implement not found page

**What does the PR do?**
- This PR Implements  `Not Found Page`

**Description of Task to be completed?**
- implement not found page
- updated routed to handle not found page
- write test for not found page
 
**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Redux.git)
-  cd into `React-Redux` and checkout to this branch `ft-implement-not-found-page-161155295`
- run the command `yarn dev` to start the application locally.
- run the command `yarn test` to ensure all tests are passing.

**What are the relevant pivotal tracker stories?**
- [#161155295](https://www.pivotaltracker.com/story/show/161155295)

**Screenshots**
- N/A
